src/test/deploy/functions/checkIam.spec.ts

```diff
@@ -0,0 +1,186 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as checkIam from "../../../deploy/functions/checkIam";
+import * as storage from "../../../gcp/storage";
+import * as rm from "../../../gcp/resourceManager";
+import * as backend from "../../../deploy/functions/backend";
+
+const STORAGE_RES = {
+  email_address: "service-123@gs-project-accounts.iam.gserviceaccount.com",
+  kind: "storage#serviceAccount",
+};
+
+const BINDING = {
+  role: "some/role",
+  members: ["someuser"],
+};
+
+const SPEC = {
+  region: "us-west1",
+  project: "my-project",
+  runtime: "nodejs14",
+};
+
+describe("checkIam", () => {
+  let storageStub: sinon.SinonStub;
+  let getIamStub: sinon.SinonStub;
+  let setIamStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    storageStub = sinon.stub(storage, "getServiceAccount").throws("Do not call");
```

Comment:
Nit: I'd prefer a clearer error like "unexpected call to storage.getServiceAccount" because it helps debug test failures.

Reply:
Fixed